.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,18 @@
+name: Cloud Formation Template
+
+on:
+  push: {branches: master}
+  pull_request: {branches: master} 
+  
+jobs:
+  cfn-lint-chk:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Cloud Formation Linter with Latest Version
+        uses: scottbrenner/cfn-lint-action@v2
```

Comment:
It is important to statically define linter version. Please install a latest version explicitly with pip.